tests/e2e_tests

```diff
@@ -23,9 +24,9 @@ do
 
     if [ -f "run_tests.bash" ]
     then
-        output="$(INFECTION_E2E_TESTS_ENV=1 bash run_tests.bash ${1:-bin/infection})"
```

Comment:
might be worth extracting `$2` into a variable like `dir`?

Reply:
done